PhysicsTools/TensorFlow/src/NTSession.cc

```diff
@@ -474,27 +444,32 @@ Status NTSession::Run(const RunOptions& run_options,
           ((measure_step_count + 1) % build_cost_model_every == 0);
```

Comment:
the upstream has the following around L452:

```diff

   args.rendezvous = run_state.rendez;
   args.cancellation_manager = &step_cancellation_manager;
-  args.runner = [this, pool](Executor::Args::Closure c) {
-    SchedClosure(pool, std::move(c));
-  };
+
   args.session_state = &session_state_;
   args.tensor_store = &run_state.tensor_store;
   args.step_container = &run_state.step_container;

```

should it be kept here?

Reply:
Yes, I added those lines again since the changes you marked in your next comment need to be reverted in order to prevent TF from managing its own thread pool again.